pkg/params/run.go

```diff
@@ -140,6 +140,14 @@ func (r *Run) UpdatePACInfo(ctx context.Context) error {
 		}
 	}
 
+	if autoConfigure, ok := cfg.Data["auto-configure-new-github-repo"]; ok {
+		r.Info.Pac.AutoConfigureNewRepo = StringToBool(autoConfigure)
```

Comment:
we should move this type of varables in a new Provider.Settings struct i think, wdyt?

Reply:
yeah .. thats the plan. I am thinking to create couple of func which could be used in operator too.
will create an issue and discuss there

Reply:
https://github.com/openshift-pipelines/pipelines-as-code/issues/932

will handle as part of this, lmk wdyt?

Reply:
sounds good,